Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:latest
+
+RUN apt update 
+RUN apt upgrade -y
+RUN apt install -y wget tar gzip unzip file
+RUN wget https://releases.mattermost.com/focalboard/0.5.0/focalboard-server-linux-amd64.tar.gz
+RUn unzip -o focalboard-server-linux-amd64.tar.gz
```

Comment:
Does this `RUn` typo throw a syntax error?

Also, really this should be one run command so the end result isnt a bunch of unnecessary layers

```dockerfile
FROM ubuntu:latest

RUN apt update \
    && apt install -y wget tar gzip unzip file \
    && wget https://releases.mattermost.com/focalboard/0.5.0/focalboard-server-linux-amd64.tar.gz \
    && tar -xvzf focalboard-server-linux-amd64.tar.gz \
    && mv focalboard /opt \
    && rm -f focalboard-server-linux-amd64.tar.gz # cleanup the unnecessary zip to reduce end container size

EXPOSE 8000

WORKDIR /opt/focalboard

CMD /opt/focalboard/bin/focalboard-server
```

Reply:
Good spot on the typo, but no, it doesn't throw an error.

As far as the "single command" is concerned, I'd be happy to condense it down to 2 commands - one for the OS and one for Focalboard?  Seems like that way we only update the OS layer if it changes, regardless of the application version?

Reply:
@jasonraimondi - I've split it out into two RUN statements instead, would appreciate your feedback?